docs/consensus_decisions.md

```diff
@@ -0,0 +1,51 @@
+# Consensus Decisions
+
+During its proceedings, the working group has reached internal consensus on a number of issues.
+This document enumerates those, and provides a reference for later actions.
+
+## 1: Include message references in the data model.
+
+**Discussion:**
+The implementers would find a way to include references anyways, but including it in the data model (standard) can make it subject to best practices.
+It’s still possible for users to do “the wrong thing” (ex: concatenation of strings/messages), but then you would find it more difficult to achieve.
```

Comment:
I suggest "It will unfortunately still be possible, but much more difficult, for users to do “the wrong thing” (ex: concatenation of strings/messages) if references are part of the data model.

Reply:
```suggestion
It will unfortunately still be possible, but much more difficult, for users to do “the wrong thing” by concatenating strings or messages.
```

The "if references are part of the data model" qualifier isn't actually valid, as explicit references are by no means required for bad usage to be possible. With MF1 I've seen plenty of examples with e.g. messages foo_1, foo_2 and foo_3 expected to be combined together after formatting.